src/Core/Style.js

```diff
@@ -112,6 +112,11 @@ class Style {
             this.point.color = color;
             this.point.opacity = opacity;
             this.point.radius = 1.5;
+        } else if (layer.type === 'circle') {
+            const { color, opacity } = rgba2rgb(layer.paint['circle-color']);
+            this.point.color = color;
+            this.point.opacity = opacity;
+            this.point.radius = layer.paint['circle-radius'];
```

Comment:
No default value ?

Reply:
if type is circle `color` and `radius` are defined.
`rgba2rgb` returns 1 by default.

Reply:
Ok !